pypesto/select/misc.py

```diff
@@ -163,3 +167,51 @@ def correct_x_guesses(
                 corrected_x_guess.append(corrected_value)
             corrected_x_guesses.append(corrected_x_guess)
     return corrected_x_guesses
+
+
+class SacessMinimizeMethod:
+    """Create a minimize method for SaCeSS that adapts to each problem.
+
+    When a pyPESTO SaCeSS optimizer is created, it takes the problem
+    dimension as input. Hence, an optimizer needs to be constructed for
+    each problem. Objects of this class act like a minimize method for model
+    selection, but a new problem-specific SaCeSS optimizer will be created
+    every time a model is minimized.
+
+    Class attributes correspond to pyPESTO's SaCeSS optimizer, and are
+    documented there.
+    """
+
+    def __init__(
+        self,
+        num_workers: int,
+        local_optimizer,
+        max_walltime_s: int,
```

Comment:
Not sure if somebody wants use wall time limits that depend on the number of parameters, but I think can be addressed when needed.

Reply:
I agree, `num_workers` too possibly, but would take more effort to benefit from this, because then there are variable numbers of processes, which probably means unused but allocated CPU on a HPC.

Parallelization of calibration of different models in a single iteration of model selection is an open point, but could help here. Anyway, agreed, will keep for a different PR.